src/hooks/common/useTimer.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
+
+export const useTimer = () => {
+  const [totalSecond, setTotalSecond] = useState(0);
+
+  useEffect(() => {
+    const timerId = setInterval(() => {
+      setTotalSecond((previousSecond) => previousSecond + 1);
+    }, 1000);
+
+    return () => {
+      clearInterval(timerId);
+    };
+  }, []);
+
+  const minute = Math.round(totalSecond / 60)
```

Comment:
p1;
floor 사용해야할 것 같은데 확인 한 번 부탁드려요!
```suggestion
  const minute = Math.floor(totalSecond / 60)
```

Reply:
의식의 흐름대로 코딩하다 반올림을 해버렸네요.. ㅎㅎ
발견해주셔서 감사합니다! 해당 부분 수정했습니다!

작업 커밋 : 4c82056e1a060a33535fad994f24b7d8578d21a5